core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -288,6 +288,10 @@
     border-bottom-right-radius: 0;
   }
 
```

Comment:
Can you please write a comment to explain why this css block? (Because we have all the CSS associated with the classes here and they begin with `.class-name` -- and this block is a bit different)

Reply:
I have added a comment block to explain the rationale behind the CSS block.